blas/tpls/KokkosBlas2_ger_tpl_spec_decl_rocblas.hpp

```diff
@@ -0,0 +1,438 @@
+                                                      , reinterpret_cast<rocblas_float_complex*>(A.data())       \
+                                                      , LDA                                                      \
+                                                      )                                                          \
+                                       );                                                                        \
+        }                                                                                                        \
+        else {                                                                                                   \
+          KOKKOS_IMPL_DO_NOT_USE_PRINTF("rocblasCgerc() requires LayoutLeft: throwing exception\n");             \
+          throw std::runtime_error("Error: rocblasCgec() requires LayoutLeft views.");                           \
+        }                                                                                                        \
+      }                                                                                                          \
+      KOKKOS_ROCBLAS_SAFE_CALL_IMPL(rocblas_set_stream(s.handle, NULL));                                         \
+      Kokkos::Profiling::popRegion();                                                                            \
+    }                                                                                                            \
+  };
+
+KOKKOSBLAS2_DGER_ROCBLAS(Kokkos::LayoutLeft,  Kokkos::Experimental::HIPSpace, true )
+KOKKOSBLAS2_DGER_ROCBLAS(Kokkos::LayoutLeft,  Kokkos::Experimental::HIPSpace, false)
+KOKKOSBLAS2_DGER_ROCBLAS(Kokkos::LayoutRight, Kokkos::Experimental::HIPSpace, true )
+KOKKOSBLAS2_DGER_ROCBLAS(Kokkos::LayoutRight, Kokkos::Experimental::HIPSpace, false)
+
+KOKKOSBLAS2_SGER_ROCBLAS(Kokkos::LayoutLeft,  Kokkos::Experimental::HIPSpace, true )
+KOKKOSBLAS2_SGER_ROCBLAS(Kokkos::LayoutLeft,  Kokkos::Experimental::HIPSpace, false)
+KOKKOSBLAS2_SGER_ROCBLAS(Kokkos::LayoutRight, Kokkos::Experimental::HIPSpace, true )
+KOKKOSBLAS2_SGER_ROCBLAS(Kokkos::LayoutRight, Kokkos::Experimental::HIPSpace, false)
+
+KOKKOSBLAS2_ZGER_ROCBLAS(Kokkos::LayoutLeft,  Kokkos::Experimental::HIPSpace, true )
+KOKKOSBLAS2_ZGER_ROCBLAS(Kokkos::LayoutLeft,  Kokkos::Experimental::HIPSpace, false)
+KOKKOSBLAS2_ZGER_ROCBLAS(Kokkos::LayoutRight, Kokkos::Experimental::HIPSpace, true )
+KOKKOSBLAS2_ZGER_ROCBLAS(Kokkos::LayoutRight, Kokkos::Experimental::HIPSpace, false)
+
+KOKKOSBLAS2_CGER_ROCBLAS(Kokkos::LayoutLeft,  Kokkos::Experimental::HIPSpace, true )
+KOKKOSBLAS2_CGER_ROCBLAS(Kokkos::LayoutLeft,  Kokkos::Experimental::HIPSpace, false)
+KOKKOSBLAS2_CGER_ROCBLAS(Kokkos::LayoutRight, Kokkos::Experimental::HIPSpace, true )
+KOKKOSBLAS2_CGER_ROCBLAS(Kokkos::LayoutRight, Kokkos::Experimental::HIPSpace, false)
```

Comment:
Replace `Kokkos::Experimental::HIPSpace` with `Kokkos::HIPSpace`, note that here you use only one macro argument to specify the layout, this is what I am referring to in the CUBLAS implementation.

Reply:
Removed "Experimental::".